bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2478,7 +2478,7 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( IsTemplatedSTLClass( name, "vector" ) ) {
+   else if ( IsTemplatedSTLClass( name, "vector" ) || IsTemplatedSTLClass( name, "ROOT::VecOps::RVec" )) {
```

Comment:
Since the `IsTemplatedSTLClass` function will not be used anymore just for STL classes, should we perhaps change its name?

Reply:
We can for sure.